webapp/src/js/actions/reports.js

```diff
@@ -124,8 +123,9 @@ angular.module('inboxServices').factory('ReportsActions',
               addSelectedReport(model);
             }
           } else {
-            if (liveList.initialised()) {
-              liveList.setSelected(model.doc && model.doc._id);
+            if (LiveList.reports.initialised()) {
+              LiveList.reports.setSelected(model.doc && model.doc._id);
+              LiveList['report-search'].setSelected(model.doc && model.doc._id);
```

Comment:
This fixes an unreported bug where when the reports tab was filtered the items never appeared selected.

Reply:
Nice catch!